frost/bip340_test.go

```diff
@@ -57,6 +57,9 @@ func TestBip340CurveDeserialize(t *testing.T) {
 		"one more than expected": {
 			input: append(serialized, 0x1),
 		},
+		"not on the curve": {
+			input: curve.SerializePoint(&Point{big.NewInt(1), big.NewInt(2)}),
```

Comment:
Should we also test it on the infinity point?

Reply:
Good point. Please see a7c405d30f15f800a7e0ff4f0a9186608e05a055.